src/nagendra/CodingExam4.java

```diff
@@ -0,0 +1,30 @@
+package nagendra;
+
+/*Program 1: WAP to Find Common Element Between Two Arrays
+input: 
+int array1 [] = [1, 2, 5, 5, 8, 9, 7, 10]
+int array2 [] = [1, 0, 6, 15, 6, 4, 7, 0]
+output: 1 7
+*/
+
+
+public class CodingExam4 {
+
+	
+	void getCommonElement(int[] input,int[] input2) {
+		for(int i=0;i<input.length;i++){
+			for(int j=0;j<input2.length;j++){
+				if(input[i]==input2[j]){
+					System.out.println("Common :"+input[i]);
```

Comment:
Just need to put break, once you get element value then you can stop iteration on second array,

Reply:
Updated as per comment

Reply:
Thanks Nagendra